groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
```

Comment:
OK I  plan on having the separate repo for packaging all the canvas Python Api library.  I will see how that goes i have to get this project get going by July 15

I am  using 'request' module , I don't know if i found any other JSON-based web service APIs. Suggest me if you have something in mind